tests/Integration/tests-full/edit-on-github/expected/page1.html

```diff
@@ -16,7 +16,7 @@
 
     <link href="_resources/css/theme.css" rel="stylesheet">
     <link href="https://docs.typo3.org/search/" rel="search" title="Search">
-                <link href="index.html" rel="prev" title="Document Title"/>
+                <link href="#" rel="prev" title="Page 1"/>
```

Comment:
Why is now "#" used for links?

Reply:
ah, shoot, I thought that was fixed in the guides. Made an issue there